app/errors.py

```diff
@@ -2,7 +2,8 @@
 
 
 class ServiceError(str, Enum):
-    ACCOUNTS_SIGNUP_FAILED = "accounts.signup_failed"
+    INTERNAL_SERVER_ERROR = "global.internal_server_error"
```

Comment:
decided to go with this for unhandled exceptions.

there is a debate against this; hiding information from possible attackers.

not 100% sure how i feel about this yet; i will probably improve this in the future to care more for that.